azure-pipelines.yml

```diff
@@ -47,12 +47,12 @@ jobs:
  - job: msrv
    pool:
      vmImage: ubuntu-latest
-   displayName: "Minimum supported Rust version: 1.40.0"
+   displayName: "Minimum supported Rust version: 1.43.0"
    dependsOn: []
    steps:
      - template: install-rust.yml@templates
        parameters:
-         rust: 1.40.0 # static-assertions (1.37+) and base64 (1.40+)
+         rust: 1.43.0 # static-assertions (1.37+) and base64 (1.40+)
```

Comment:
Could you replace the comment here too now that the reason for the MSRV is different.

Reply:
No problem! :-)